src/ethereum/EnigmaContractProductionWriterAPI.js

```diff
@@ -134,15 +134,15 @@ class EnigmaContractProductionWriterAPI extends EnigmaContractWriterAPI {
       const blockNumber = await utils.getEthereumBlockNumber(this.w3());
 
       const resolveLogic = async () => {
-        let deployedEvents = await this._parsePastEvents(
+        const deployedEvents = await this._parsePastEvents(
```

Comment:
Please set a parameter `filter` and past to the call to `this._parsePastEvents` (as done in the `commitReceipt` function) now that both filters are the same.

Reply:
Thanks, will do.